patience-deck-pl.ts

```diff
@@ -0,0 +1,363 @@
+        <location filename="../qml/pages/about/AboutPage.qml" line="131"/>
+        <source>License</source>
+        <extracomment>Section that mentions license(s) of Patience Deck</extracomment>
+        <translation>Licencja</translation>
+    </message>
+    <message id="patience-la-free_software_and_redistribution">
+        <location filename="../qml/pages/about/AboutPage.qml" line="139"/>
+        <source>Patience Deck is free software, and you are welcome to redistribute it under certain conditions. This software comes with ABSOLUTELY NO WARRANTY. Tap the button below for more details.</source>
+        <translation>Patience Deck to darmowe oprogramowanie, które możesz rozpowszechniać pod pewnymi warunkami. To oprogramowanie nie jest objęte ABSOLUTNIE ŻADNĄ GWARANCJĄ. Kliknij przycisk poniżej, aby uzyskać więcej informacji.</translation>
+    </message>
+    <message id="patience-bt-licenses">
+        <location filename="../qml/pages/about/AboutPage.qml" line="145"/>
+        <source>Licenses</source>
+        <translation>Licencje</translation>
+    </message>
+    <message id="patience-se-experimental">
+        <location filename="../qml/pages/about/AboutPage.qml" line="153"/>
+        <source>Experimental</source>
+        <extracomment>Section for enabling experimental features</extracomment>
+        <translation>Funkcje eksperymentalne</translation>
+    </message>
+    <message id="patience-la-show_all_games">
+        <location filename="../qml/pages/about/AboutPage.qml" line="158"/>
+        <source>Show all games</source>
+        <translation>Pokaż wszystkie gry</translation>
+    </message>
+    <message id="patience-de-list_unsupported_games">
+        <location filename="../qml/pages/about/AboutPage.qml" line="160"/>
+        <source>List also unsupported games in game selection</source>
+        <translation>Wyświetl również nieobsługiwane gry w wyborze gier</translation>
+    </message>
+    <message id="patience-he-thanks_to">
+        <location filename="../qml/pages/about/AboutPage.qml" line="189"/>
+        <source>Thanks to</source>
+        <translation>Podziękowania dla</translation>
+    </message>
+    <message id="patience-de-aisleriot_authors">
+        <location filename="../qml/pages/about/AboutPage.qml" line="191"/>
+        <source>GNOME Aisleriot authors</source>
+        <translation>autorów GNOME Aisleriot</translation>
+    </message>
+    <message id="patience-se-translator-information">
+        <location filename="../qml/pages/about/AboutPage.qml" line="204"/>
+        <source>Aisleriot translator information for language</source>
+        <extracomment>Section to show Aisleriot translator/translation information, substitute word language with the name of the language of the translation. Original is pretty generic on purpose, feel free to translate this to something works for the result</extracomment>
+        <translation>Informacje o tłumaczu Aisleriot dla języka</translation>
```

Comment:
I checked what online translator gives me about this and it seems that you translated this quite literally. The intention of mine was to say for example "Aisleriot translator information for Polish", so word language is substituted with language name. It can be also something different as the text below this can vary greatly from one language to another.